src/index.ts

```diff
@@ -1,5 +1,50 @@
+import * as fs from "fs";
+import * as path from "path";
+import {AvroToTypescriptCompiler} from "./components/AvroToTypescriptCompiler";
+import {ConsoleHelper} from "./helpers/ConsoleHelper";
+
 export * from "./components/AvroToTypescriptCompiler";
 export * from "./components/avroToTypescript/EnumConverter";
 export * from "./components/avroToTypescript/PrimitiveConverter";
 export * from "./components/avroToTypescript/AvroSchemaConverter";
 export * from "./components/avroToTypescript/RecordConverter";
+
+switch (process.argv[2]) {
```

Comment:
I think it is easier and better to use already finished command line interfaces.
Something like [args](https://www.npmjs.com/package/args) maybe?

Reply:
Wasn't sure if I could add additional dependencies, will look into it then :)